deployment/ccip/changeset/testhelpers/test_assertions.go

```diff
@@ -19,6 +19,8 @@ import (
 	commonutils "github.com/smartcontractkit/chainlink-common/pkg/utils"
 	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
 
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+	changeset2 "github.com/smartcontractkit/chainlink/deployment/common/changeset"
```

Comment:
Rename import to changesetcommon or cscommon or whatever

```suggestion
	cscommon "github.com/smartcontractkit/chainlink/deployment/common/changeset"
```